### DISCUSSION TITLE
Is it possible to support 東風戦 online by Mortal?

### DISCUSSION THREAD

**Discussion by ShadowFiend2015** [Equim-chan/Mortal#53] [Q&A] [answered]
I'm a player in  Mahjong Soul(雀魂), for some of reason I don't have much time to play hanchan(半荘). I wonder if developer could support 東風戦 online(https://mjai.ekyu.moe/), thx a lot.
By the way, is it possible to support 3p? thx again.

**Comment by Equim-chan** [comment, marked as answer]
It is theoretically possible, but it is not trivial work because it will break a lot of code. Currently there are no plans to support tonpuu.

As for 3p, the answer would be impossible because it's a completely different game.
